op-conductor/consensus/mocks/Consensus.go

```diff
@@ -0,0 +1,539 @@
+// Code generated by mockery v2.28.1. DO NOT EDIT.
```

Comment:
[nit] consensus.go

Reply:
These names seem to be standard behavior from mockery, and is consistent with how other mocks are generated within the repo, example:

* https://github.com/ethereum-optimism/optimism/blob/develop/op-node/p2p/gating/mocks/ExpiryStore.go
* https://github.com/ethereum-optimism/optimism/blob/develop/op-service/txmgr/mocks/TxManager.go

That said, happy to try to change all to snake case if necessary.

If not, please help add it to merge queue